src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -42,12 +44,16 @@ public class StrykerOptions
         public string OptimizationMode { get; set; }
         public IEnumerable<string> TestProjects { get; set; }
 
+        public string CurrentBranchCanonicalName { get; set; }
```

Comment:
This is not an option, can the dashboard reporter not just use the branch provider by itself?